app/metrics.yaml

```diff
@@ -4,6 +4,21 @@
 
 $schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
 
+events:
+  app_opened:
+    type: event
+    description: >
+      A User opened the app
+    extra_keys:
+      source: "The source from which the app was opened"
```

Comment:
Ah I see the higher detail later in the docs! I think the goal of Glean is to avoid having to write those docs anymore, so put all that detail and values and explanation right here in the yaml file itself.

Reply:
:+1: whenever possible